src/ActivityResource.php

```diff
@@ -113,7 +113,7 @@ protected function formatValue(mixed $value): string
 
         if (
             strpos($value, 'T') === 10
-            && strpos($value, 'Z') === 26
+            && strpos($value, 'Z') === 19
```

Comment:
This is to catch JSON formatted datesm such as: `2019-02-01T03:45:27.612584Z`.

However, given the call inside just goes straight to `parse()`, this could be improved by making the check just look for the presence of `T` or `Z`, no matter where it is in the string.